src/routes/members/+page.js

```diff
@@ -1 +1 @@
-export let csr = true;
\ No newline at end of file
+export let csr = true;
```

Comment:
Waarom is dit weer toegevoegd? Misschien handig om er een comment bij te zetten?

Reply:
Ik liet gister die error zien op de members page waarvan ik dacht dat export csr er misschien mee te maken had alleen was dat niet het geval.